test/Microsoft.ML.Tests/Transformers/ConcatTests.cs

```diff
@@ -62,13 +63,13 @@ ColumnType GetType(Schema schema, string name)
             t = GetType(data.Schema, "f4");
             Assert.True(t is VectorType vt4 && vt4.ItemType == NumberType.R4 && vt4.Size == 0);
 
-            data = ColumnSelectingTransformer.CreateKeep(Env, data, new[] { "f1", "f2", "f3", "f4" });
+            data = ML.Transforms.SelectColumns("f1", "f2", "f3", "f4").Fit(data).Transform(data);
```

Comment:
I'm becoming more sympathetic to those that consider #2354 a priority @artidoro and @glebuk. 😆